menpo/base.py

```diff
@@ -210,9 +191,15 @@ def set_target(self, new_target):
         ----------
         new_target : :map:`PointCloud`
             The new target that this object should try and regenerate.
+
+        Returns
+        -------
+        self : :map:`Targetable`
+            A reference to this instance.
         """
         self._target_setter_with_verification(new_target)  # trigger the update
         self._sync_state_from_target()  # and a sync
+        return self
```

Comment:
Because otherwise writing all the AAM code is a huge annoyance because every time we set_target we need another line just to grab the target we just set. It was just tons cleaner to be able to do set_target(blah).target